### DISCUSSION TITLE
キーソケットの剥離の直し方について（Keyball61）

### DISCUSSION THREAD

**Discussion by ntakaaki** [Yowkees/keyball#393] [Q&A]
お世話になっております。
Keyball61の左手側（トラボなし）で青で囲ったキーソケットののランドが剥がれてしまいました。
こちらの直し方をご教示いただけますか。
回路図を見てみたのですが，見方がよく分からずどことどこをジャンパすればよいのかわかりませんでした。

よろしくお願いいたします。

![keyball61_hakuri](https://github.com/Yowkees/keyball/assets/1164805/94cb28a0-f9df-473c-a8f1-d0d4dae47096)

**Comment by Ryo624** [comment]
ご連絡ありがとうございます。

写真に示してあります赤点どうしをエナメル線やゴム被覆導線で接続していただければ復活できると思います。
![276890897-94cb28a0-f9df-473c-a8f1-d0d4dae47096](https://github.com/Yowkees/keyball/assets/143698349/952fb50d-0066-428d-b757-ffb3dc782617)

**Reply by ntakaaki** [reply]
ありがとうございます！
無事解決してキーの入力ができるようになりました！

**Reply by Ryo624** [reply]
無事に解決していただきましてありがとうございます！
末永くご愛用していただければ幸いです。
